pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -485,16 +473,23 @@ LANGUAGE c AS 'MODULE_PATHNAME', '@FUNCTION_NAME@';
 fn index_size(index_name: &str) -> Result<i64> {
     // Fetch the OID of the index using its name
     let oid_query = format!(
-        "SELECT oid FROM pg_class WHERE relname = '{}_bm25_index' AND relkind = 'i'",
+        "SELECT oid, relfilenode FROM pg_class WHERE relname = '{}_bm25_index' AND relkind = 'i'",
         index_name
     );
 
-    let index_oid = Spi::get_one::<pg_sys::Oid>(&oid_query)?
-        .ok_or_else(|| anyhow::anyhow!("Index '{}' not found", index_name))?
-        .as_u32();
+    let database_oid = crate::MyDatabaseId();
+    let (index_oid, relfilenode) = match Spi::get_two::<pg_sys::Oid, pg_sys::Oid>(&oid_query) {
+        Ok((Some(index_oid), Some(relfilenode))) => (index_oid, relfilenode),
+        Ok((None, _)) => panic!("no oid for index '{index_name}_bm25_index' in schema_bm25"),
+        Ok((_, None)) => {
+            panic!("no relfilenode for index '{index_name}_bm25_index' in schema_bm25")
+        }
+        Err(err) => panic!("error looking up index '{index_name}_bm25_index': {err}"),
+    };
```

Comment:
Similar to above -- lets avoid SPI here and use the direct_function_call API to get the relation oid from `pg_sys::regclassin`